clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -2751,6 +2751,7 @@ static void RenderFloatingPointOptions(const ToolChain &TC, const Driver &D,
   // If one wasn't given by the user, don't pass it here.
   StringRef FPContract;
   StringRef LastSeenFfpContractOption;
+  StringRef LastFpContractOverrideOption;
```

Comment:
`Overridden` seems more appropriate, but the variable name will become too long.
I wonder whether `Last` can be removed. It's clear from the type that this can only hold one option name.